packages/react/src/__tests__/events.test.tsx

```diff
@@ -51,13 +50,17 @@ it('should assign correct path to an event target', () => {
               key: 'container',
               props: {
                 ['data-testid']: 'button',
-                onClick: event(jest.fn()),
+                onClick: {
+                  $: 'event',
+                },
```

Comment:
We're defining client-side DataState in unit tests and here, `handler` is not present (functions are not part of JSON after stringifying them). So, we're not going to use the `event()` helper, but rather define some actual JSON.